assets/js/base/context/cart-checkout/payment-methods/actions.js

```diff
@@ -80,3 +86,27 @@ export const success = ( { billingData, paymentMethodData } ) => ( {
 	billingData,
 	paymentMethodData,
 } );
+
+/**
+ * Used to dispatch an action for updating a registered payment method in the
+ * state.
+ *
+ * @param {Object} paymentMethod Payment method to register.
+ * @return {Object} An action object.
+ */
+export const setRegisteredPaymentMethod = ( paymentMethod ) => ( {
+	type: SET_REGISTERED_PAYMENT_METHOD,
+	paymentMethod,
+} );
+
+/**
+ * Used to dispatch an action for updating a registered express payment
+ * method in the state.
+ *
+ * @param {Object} paymentMethod Payment method to register.
+ * @return {Object} An action object.
+ */
+export const setRegisteredExpressPaymentMethod = ( paymentMethod ) => ( {
```

Comment:
Should I be testing express payment flows? I was testing with this area at the bottom of checkout, this is regular payment right (not "express")? 

<img width="694" alt="Screen Shot 2020-03-26 at 3 03 15 PM" src="https://user-images.githubusercontent.com/4167300/77602682-eceaff00-6f72-11ea-8862-11274ba06ff8.png">

Reply:
Ya the screenshot is regular payment methods. There are no explicit tests for express payment flows other than if the payment method icons appeared in express payment container, then the this code is covered by testing.  I verified that in my testing so we should be good.